src/server/pkg/testutil/cmds.go

```diff
@@ -105,7 +105,7 @@ export -f yes # use in subshells too
 
 which match >/dev/null || {
 	echo "You must have 'match' installed to run these tests. Please run:" >&2
-	echo "  cd ${GOPATH}/src/github.com/pachyderm/pachyderm/ && go install ./src/testing/match" >&2
+	echo "  go install ./src/testing/match" >&2
```

Comment:
How would you feel about `go install github.com/pachyderm/pachyderm/src/testing/match`? It's shorter and less side-effectful than `cd ...` but still works when you're not in the top-level `pachyderm/pachyderm` dir.

Reply:
I suppose the one minor risk with that is that we make changes to `match`, and someone trying to run older tests mistakenly installs the latest version of `match`. But I think it's minor enough that this makes sense.